cmd/ctrlc/root/sync/kubernetes/vcluster.go

```
@@ -0,0 +1,188 @@
+	"github.com/loft-sh/log"
+	"github.com/loft-sh/vcluster/pkg/cli/find"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	vclusterKind = "VCluster"
+)
+
+func deepClone(src map[string]interface{}) (map[string]interface{}, error) {
```

golangci-lint (1.64.8)

func `deepClone` is unused

(unused)